### DISCUSSION TITLE
Is it possible to impose a homogeneous isotropic turbulence box with unequal length and width at inlet?

### DISCUSSION THREAD

**Discussion by SKT-Bush** [AMReX-Combustion/PeleC#741] [Q&A]
I am trying to simulate a spatially developing turbulent boundary layer on a flat plate. In order to reduce the size of the calculation area, I want to impose a homogeneous isotropic turbulence box with a width of L and a height of 1/2L at inlet. Can this be achieved?

**Comment by marchdf** [comment]
Hi, thanks for reaching out! Am I understanding correctly that you want to have a turbulent inflow boundary condition? If so, you can look at this case: https://github.com/AMReX-Combustion/PeleC/tree/development/Exec/Production/ChallengeProblem. It initializes 4 turbulent jets at the zhi BC. You could easily adapt that case so that you use the turbulent fluctuations to have a turbulent inlet.
